.github/workflows/enos-verify-stable.yml

```diff
@@ -4,10 +4,12 @@ on:
   repository_dispatch:
     types:
       - enos-verify-stable
+      - enos-verify-stable::*
 
 jobs:
   enos-verify-stable:
     name: Enos verify stable artifact
+    if: ${{ startsWith(github.event.client_payload.payload.branch, 'release/') }}
```

Comment:
So this only runs post-build workflow on release branches?

Reply:
sorry just saw this, yes only runs post-build workflow on release branches, as I thought running it on `main` might cause resource bottlenecks for Enos infra